components/artist_page_cta/view.coffee

```diff
@@ -53,10 +53,18 @@ module.exports = class ArtistPageCTAView extends Backbone.View
       afterAuthPath: @afterAuthPath
     @$(".artist-page-cta-overlay__register input[name='name']").focus()
     @$('.artist-page-cta-overlay__close').on 'click', @closeOverlay
+    @disableScroll()
+
+  disableScroll: ->
+    $('#main-layout-container').addClass('artist-page-cta__no-scroll')
```

Comment:
Can you do a little gif of this actually? That BEM'ed class is confusing me. I guess it isn't going to inherit its parent class, but its still a bit weird. Maybe better to make that no-scroll class in the `main_layout` component instead...